integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala

```diff
@@ -128,6 +128,10 @@ object IndexServer extends ServerInterface {
   def getCount(request: IndexInputFormat): LongWritable = {
     doAs {
       val sparkSession = SparkSQLUtil.getSparkSession
+      var currentUser: String = null
+      if (!request.isFallbackJob && Server.getRemoteUser != null) {
+        currentUser = Server.getRemoteUser.getShortUserName
```

Comment:
what happens if the RemoteUser is null?

Reply:
Null check for Server.getRemoteUser removed